specification/_global/search/_types/sort.ts

```diff
@@ -104,12 +108,5 @@ export enum SortMode {
 
 export enum SortOrder {
   asc = 0,
-  desc = 1,
-  /** @codegen_name Document */
-  _doc = 2
-}
-
-export enum SortSpecialField {
-  _score = 0,
-  _doc = 1
```

Comment:
This enum is not needed. If users want to use the default order for `_score` and `_doc` then they should use the `Field` member of `SortCombinations`. And otherwise use the `_score` and `_doc` variants of `SortOptions` whose properties are different from that of a `FieldSort`.